src/partition/repart.c

```diff
@@ -2465,9 +2476,13 @@ static int partition_encrypt(
                 *ret_volume = TAKE_PTR(vol);
 
         return 0;
+#else
+        return log_error_errno(-ENOENT, "libcryptsetup not found, cannot encrypt: %m");
```

Comment:
Use `SYNTHETIC_ERRNO(EOPNOTSUPP)` rather than `-ENOENT` please.

Also, the msg is misleading, support for libcryptsetup is compiled out, it's not just that the lib is not found.